pages/integrations/github.md.erb

```diff
@@ -0,0 +1,58 @@
+# GitHub
+
+Buildkite can connect to your GitHub repository and use the Commit Status API to update the status of commits in Pull Requests.
+
+To complete this integration, you will need admin privileges for your GitHub repository.
+
+<%= toc %>
+
+## Connecting Buildkite and GitHub
+
+To connect your GitHub account open Buildkite’s Personal Settings, choose [Connected Apps](https://buildkite.com/user/connected-apps) then click the GitHub Connect button:
+
+<%= image "personal-settings.png", width: 1467/2, height: 565/2, alt: "Screenshot of the Buildkite Connected Apps screen" %>
+
+You'll be prompted to give permission for Buildkite to post status
+updates, and then redirected back to your Connected Apps page.
+
+## Setting up a new pipeline for a GitHub repository
+
+1. Create a new pipeline on Buildkite, using your GitHub repository URL in the form `git@github.com:your/repo`.
+
+    <%= image "new-pipeline.png", width: 1464/2, height: 744/2, alt: "Screenshot of adding a new pipeline " %>
+
+2. Make sure to add at least one command to your pipeline, before clicking next.
+
+3. Follow the instructions on the page to set up a webhook:
+
+    1. Add a new webhook in Github.
+    2. Paste in the the provided webhook URL.
+    3. Select which events trigger the webhook.
+
+## Branch configuration and Settings
+
+You can edit the GitHub settings for each pipeline from the pipeline’s settings page:
```

Comment:
Given that this section's header is branch config, what do you think about linking off to the conditionals page or adding a sentence talking about how you can do more advanced config using conditionals in your pipeline?

Reply:
That makes sense, but does drag me into a (much) longer conversation about whether these are docs for the GitHub integration or whether these are docs for setting up Buildkite on GitHub, and then onto consistency with the other integrations.

But it is a good conversation to have! :-D 

For now I'll add something.

Reply:
Ah yeah, good point. Keen to talk those ideas through!